src/configs/base_keyspecs.c4m

```diff
@@ -1640,6 +1640,22 @@ Platform passed when performing `docker build`, if any.
 """
 }
 
+keyspec DOCKERFILE_VCTL_REL_PATH {
+    kind:             ChalkTimeArtifact
+    type:             string
+    standard:         true
+    system:           false
+    since:            "0.4.13"
+    shortdoc: "Relative path of the Dockerfile to the VCS' root."
+    doc: """
```

Comment:
nit:

```suggestion
    shortdoc:         "Relative path of the Dockerfile to the VCS' root."
    doc:              """
```

many other keys align their values

would be nice to eventually get autoformatter but for now we try to do it manually